src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php

```diff
@@ -83,6 +83,7 @@ protected function findTemplate($template, $throw = true)
                 $template = $this->parser->parse($template);
                 $file = $this->locator->locate($template);
             } catch (\Exception $e) {
+            } catch (\Throwable $e) {
```

Comment:
same reasoning here than for the DelegatingLoader. An Error can only happen for a bug in the original code, not for unsupported template notations, and bugs should not be hidden